src/theme/base.ts

```diff
@@ -32,19 +32,15 @@ declare module "@material-ui/core/styles/createPalette" {
     status: {
       running: {
         text: string;
-        background: string;
```

Comment:
If don't require `background` completely then we can change the interface to be like `running: string`. So that `text: string` will be omitted. May we change it?

Reply:
cc: @arkajyotiMukherjee

Reply:
Agreed can be changed.